common/current_preset.h

```diff
@@ -271,10 +271,30 @@ class CurrentPreset {
 
   void SetPreset(int preset) {
     Clear();
+    #ifdef SAVED_PRESET
+    SavePreset(preset);
+    #endif
     LOCK_SD(true);
     if (!Load(preset)) Set(preset);
     LOCK_SD(false);
   }
+
+  bool SavePreset(int preset) {
+    if (preset >= 0 && preset <= current_config->num_presets) {
+      STDOUT.println("Saving Current Preset");
+      LOCK_SD(true);
+      FileReader c;
+      LSFS::Remove("savedpreset.ini");
+      c.Create("savedpreset.ini");
+      char value[30];
+      itoa(preset, value, 10);
+      c.write_key_value("preset", value);
+      itoa(dynamic_mixer.get_volume(), value, 10);
+      c.write_key_value("volume", value);
```

Comment:
Not sure if it makes sense to save the volume if there is no way to change it...

Reply:
Yes, but once this hopefully gets merged, was going to propose a volume menu.

Reply:
I'm not a big fan of menues, but as long as it can be enabled/disabled with defines that might be ok.